drv/vsc7448/src/serdes1g.rs

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::{spi::Vsc7448Spi, VscError};
+use vsc7448_pac::Vsc7448;
+
+pub enum Mode {
+    Sgmii,
+}
+pub struct Config {
+    ob_amp_ctrl: u32,
+    ob_idle: u32,
+    cmv_term: u32,
+    test_mode: u32,
+    test_pattern: u32,
+    mode_100fx: bool,
+    ena_dc_coupling: bool,
+    des_phs_ctrl: u32,
+    cpmd_sel: u32,
+    mbtr_ctrl: u32,
+    des_bw_ana: u32,
+    ena_lane: bool,
+}
+/// Based on `jr2_sd1g_cfg`
+impl Config {
+    pub fn new(m: Mode) -> Self {
+        match m {
+            Mode::Sgmii => Self {
+                ob_amp_ctrl: 12,
```

Comment:
Where would I check these against, were I to wish to?

Reply:
There's a docstring at the [top of the struct](https://github.com/oxidecomputer/hubris/pull/391/files/1d809c08e4e9703dd8b811394b6d8faaed3ec4be#diff-0de703a68cb440b6446654586e5e97a06f16a35eaf2424b390d395664390f1c7R25)